pages/knowledge/colours-and-contrast/README.md

```diff
@@ -0,0 +1,9 @@
+---
+navigation_title: "Colours & contrast"
+position: 1
+changed: "2018-12-26"
+---
+
+# Colours and contrast
+
+**Accessibility goes far beyond functionality and code. When it comes to visual appearances, colours require special consideration. The meaning of a certain colour as well as the contrast between multiple colours directly affect how well a person can perceive and understand what is on screen. Colour palettes should therefore be composed carefully and according to some basic rules.**
```

Comment:
I'd love to read something like the following here, too:

> Visual content (like text, icons, form controls, charts, etc.) needs to be easily perceivable. As such, it must clearly stand out visually from its background.

This gives a very fast understanding, what typical elements we are talking about (otherwise it feels to me like a novice could think we are talking about a very abstract design concept here).

Reply:
**To Do:** simplify text.

Reply:
New paragraph

>>>
Visual elements like for example text, icons, input fields and charts need to be easily perceivable. On the most basic level that means that they have to clearly stand out from their background. But it’s also important that they can be easily distinguished from similar elements around them – for example links embedded in a text paragraph. The visual contrast between different colours is a key criteria in this context. But colour contrast also has its limitations.
>>>